nginx_config_reloader/__init__.py

```diff
@@ -210,7 +218,12 @@ def apply_new_config(self):
         else:
             self.remove_error_file()
 
-        self.reload_nginx()
+        if self.nats_client:
```

Comment:
This way, all nginx instances will be reloaded, but not every instance will have copied over the nginx files from source to dest right?

Reply:
Good point. Fixed in c95ee23